### PR TITLE
Task/csi 4888 wait for crds

### PR DESCRIPTION
**Pull request by ELENAGER**

### CODE REVIEW

File: volume-group-operator/controllers/volumegroup_controller.go [hunk 1]
```diff
@@ -18,19 +18,31 @@ package controllers
...
 	grpcClient "github.com/IBM/volume-group-operator/pkg/client"
 )
 
+var volumeGroupLog = log.Log.WithName("volumegroup_controller")
+
+const (
+	VolumeGroup               = "VolumeGroup"
+	VolumeGroupClass          = "VolumeGroupClass"
+	VolumeGroupContent        = "VolumeGroupContent"
+)
```
Comment (matancarmeli7) [inline volume-group-operator/controllers/volumegroup_controller.go:44]: can you move it settings file in this package?

File: volume-group-operator/controllers/volumegroup_controller.go [hunk 2]
```diff
@@ -68,3 +86,53 @@ func (r *VolumeGroupReconciler) SetupWithManager(mgr ctrl.Manager, cfg *config.D
...
+		// return errors other than NoMatch
+		if !meta.IsNoMatchError(err) {
+			logger.Error(err, "got an unexpected error while waiting for resource", "Resource", resourceName)
+
+			return err
+		}
+		logger.Info("resource does not exist", "Resource", resourceName)
+		time.Sleep(5 * time.Second)
+	}
+}
```
Comment (matancarmeli7) [inline volume-group-operator/controllers/volumegroup_controller.go:138]: can you add a new line?
image

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: volume-group-operator/controllers/volumegroup_controller.go (hunk 2 above)
Comment (ELENAGER) [reply volume-group-operator/controllers/volumegroup_controller.go]: Done

File: volume-group-operator/controllers/volumegroup_controller.go (hunk 1 above)
Comment (ELENAGER) [reply volume-group-operator/controllers/volumegroup_controller.go:42]: Following our conversation, I will move these consts into settings,go file that is waiting to be merged in PR https://github.com/IBM/ibm-block-csi-operator/pull/294

**Pull request merged**